### DISCUSSION TITLE
Blockchain

### DISCUSSION THREAD

**Discussion by tripleo1** [gildas-lormeau/SingleFile#797] [General]
> Enable the option "Misc. > add proof of existence" to prove the existence of saved pages by linking the SHA256 of the pages into the blockchain.

What does this mean and how does it work?

**Comment by gildas-lormeau** [comment]
You can find more information about this feature by clicking on the "?" icon at the right of the corresponding label in the options page  (cf. setting `Misc > add proof of existence`) or in the help page of the extension. Here is below what the documentation says:

> Check this option to create a worldwide proof of the existence of the page you want to save.
> 
> **What is a proof of existence (data anchoring)?**
> Data anchoring consists in building a time-stamped proof of existence for a data by linking it to a tamper resistant and time-stamped blockchain. Data anchoring implementation relies on the resilience and immutability of the Bitcoin blockchain to provide the best possible security level
> 
> **How does this protect my data?**
> The anchoring mechanism only handles data impressions. Your data remains where you calculate the fingerprints, i.e. in the browser. Their confidentiality is totally preserved.
> 
> The day after your backup you can get freely the proof receipt here: [https://gildas-lormeau.github.io/singlefile-woleet/index.html](https://gildas-lormeau.github.io/singlefile-woleet/index.html). A proof receipt will be used to verify the validity of the evidence
> 
> More information [doc.woleet.io](doc.woleet.io) 
>

**Reply by gildas-lormeau** [reply]
You can also find some additional info in the post that announced the feature: https://blog.woleet.io/woleet-singlefile-extension-bitcoin-proof/

**Comment by mariomadproductions** [comment]
As far as I can tell, this just proves that the file existed at at point in time, it doesn't prove that the file is genuine.
